CONTRIBUTING.md

```diff
@@ -39,12 +37,9 @@ Feel free to check out the `#discussion`/`#development` channels on our [Slack](
 
 ## Developing
 
-**Note:** Versions `< 5.1.10` can't be built.
+*Node*: Check that Node is [installed](https://nodejs.org/en/download/) with version 6 and up. You can check this with `node -v`.
 
-Babel is built for Node 6 and up but we develop using Node 8 and yarn. You can check this with `node -v`.
-
-Make sure that Yarn is installed with version >= `0.28.0`.
-Installation instructions can be found here: https://yarnpkg.com/en/docs/install.
+*Yarn*: Make sure that Yarn is [installed](https://classic.yarnpkg.com/en/docs/install) with version >= `0.28.0`.
```

Comment:
May worth referring to http://gnuwin32.sourceforge.net/packages/make.htm for Windows developers.

Reply:
yeah I'm trying to make this work on windows lol and hitting the make issue 😛